core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts

```diff
@@ -54,15 +54,13 @@ angular.module('oppia').directive('storyEditor', [
       controller: [
         '$scope', '$window', 'StoryEditorStateService', 'StoryUpdateService',
         'UndoRedoService', 'StoryEditorNavigationService',
-        'WindowDimensionsService',
-        'EVENT_VIEW_STORY_NODE_EDITOR', '$uibModal',
+        'WindowDimensionsService', '$uibModal',
```

Comment:
This doesn't looked sorted? Please fix this.

cc @Hudda Why did the linter not catch this?

Reply:
Umm... I didn't reorder this, just removed an event. Is there a linter check for this? This is a problem with quite a few files. Although I'm not sure how urgent this is considering the codebase is going to be migrated to Angular anyway

Reply:
> Why did the linter not catch this?

@Showtim3 We do not have lint check for this yet, but it will be implemented in a few days. Thanks!